tests/text/pipelines/test_multifield_bimpm.py

```diff
@@ -96,46 +145,60 @@ def pipeline_yaml(tmpdir):
             ],
         },
     }
+    pipeline_yaml = tmpdir.join("pipeline.yml")
+    with pipeline_yaml.open("w") as f:
+        yaml.safe_dump(pipeline_dict, f)
 
-    yaml_file = tmpdir.join("pipeline.yml")
-    with yaml_file.open("w") as f:
-        yaml.safe_dump(yaml_dict, f)
-
-    return str(yaml_file)
+    return str(pipeline_yaml)
 
 
 @pytest.fixture
 def trainer_yaml(tmpdir):
-    yaml_dict = {
-        "iterator": {
-            "batch_size": 3,
-            "sorting_keys": [["record1", "num_fields"]],
-            "type": "bucket",
-        },
+    trainer_dict = {
+        # "iterator": {
+        #     "batch_size": 3,
+        #     "sorting_keys": [
+        #         [
+        #             "record1",
+        #             "num_fields" if request.param == "multifield" else "num_tokens",
+        #         ]
+        #     ],
+        #     "type": "bucket",
+        # },
+        # Missing: batch_size; max_instances_in_memory; cache_instances;
```

Comment:
`batch_size` and `cache_instances` are properly exposes in [this PR](https://github.com/recognai/biome-text/pull/124/files#diff-d550678364915bd06d14ae55044d0fbbR91). Also,  `max_instances_in_memory` is auto-calculated, and should be since could be a failing point in training process.

Reply:
Got it